src/Gui/Gui.cpp

```diff
@@ -672,21 +673,20 @@ bool Gui::ShowMenu(Game& game)
 	{
 		if (ImGui::BeginMenu("Load Island"))
 		{
-			constexpr std::array<std::pair<std::string_view, std::string_view>, 6> RegularIslands = {
-			    // clang-format off
-			    std::pair {"Land 1", "Land1.txt"},
-			    std::pair {"Land 2", "Land2.txt"},
-			    std::pair {"Land 3", "Land3.txt"},
-			    std::pair {"Land 4", "Land4.txt"},
-			    std::pair {"Land 5", "Land5.txt"},
-			    std::pair {"God's Playground", "LandT.txt"},
-			    // clang-format on
-			};
-			constexpr std::array<std::pair<std::string_view, std::string_view>, 3> PlaygroundIslands = {
-			    std::pair {"Two Gods", "TwoGods.txt"},
-			    std::pair {"Three Gods", "ThreeGods.txt"},
-			    std::pair {"Four Gods", "FourGods.txt"},
-			};
+			auto& resources = Locator::resources::ref();
+			auto& levelsManager = resources.GetLevels();
+			auto campaigns = std::vector<entt::resource_handle<const Level>>();
+			auto playgrounds = std::vector<entt::resource_handle<const Level>>();
+			levelsManager.Each([&campaigns, &playgrounds](entt::id_type id, entt::resource_handle<const Level> level) {
```

Comment:
Warnings here

Reply:
Is this fixed? I'm not seeing it but I bet it's unused `id`

Reply:
You can see warnings in `Files Changed`